src/catalog/product-content.md

```diff
@@ -1,8 +1,8 @@
 ---
-title: Content
+title: Product Content
 ---
 
-The Content section is used to enter and edit the main product description that appears on the product page. The short description can be used in most RSS feeds, and might also appear in catalog listings, depending on theme.
+The _Content_ section is used to enter and edit the main product description that appears on the product page. The short description can be used in most RSS feeds, and might also appear in catalog listings, depending on theme.
```

Comment:
if there is a relevant Theme topic, suggest linking "theme" so they can get info on the theme settings.

Reply:
Good suggestion. I will looks and see if we have a good one for linking.